imager/imager/tests.py

```diff
@@ -0,0 +1,80 @@
+from django.test import TestCase, Client
+from django.core import mail
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+from imagerprofile.models import ImagerProfile
+import factory
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = User
+        django_get_or_create = ('username', 'password', 'is_active')
+
+    username = factory.Sequence(lambda n: "User%03d" % n)
+    password = 'secret'
+    is_active = True
+
+
+class ProfileFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = ImagerProfile
+        django_get_or_create = ('user',)
+
+    user = factory.SubFactory(UserFactory)
+
+
+class LoginTestCase(TestCase):
+    # def setUp(self):
+    #     """set up basic client"""
+    #     self.client = Client()
+
+    def loginHelper(self, username, password):
+        return self.client.post('/login', {'username': username,
+                                           'password': password})
+
+    def test_home(self):
+        """test that home page is available to logged out user"""
+        response = self.client.get(reverse('home'))
+        self.assertEqual(response.status_code, 200)
+
+    def test_register(self):
+        """test registration is available, and redirects on post"""
+        username = 'sally'
+        email = 'sally@sally.com'
+        password = 'secret'
+        response = self.client.get(reverse('registration_register'))
+        self.assertEqual(response.status_code, 200)
+        response = self.client.post(
+            reverse('registration_register'), {
+                'username': username,
+                'email': email,
+                'password': password
+            })
+        self.assertEqual(response.status_code, 200)
```

Comment:
Can you verify at this point that the registration email was sent?  You *should*.